prefix_tags/prefix_tags_config.c

```diff
@@ -78,9 +43,39 @@ int tags_parse_ip_prefix(const char *ip_prefix, ip_addr_t *addr, uint32_t *prefi
    return 0;
 }
 
+/** Dealloc ipps_network_list_t
+ * Dealloc struct ipps_network_list_t, opposite of load_network
+ * @param[in] network_list Pointer to network_list structure
+ * @return void
+ */
+void destroy_networks(ipps_network_list_t *network_list)
```

Comment:
From my limited point-of-view, this looks like that it should be part of `Nemea-Framework/unirec/ip_prefix_search.c`